nullaway/src/test/java/com/uber/nullaway/jspecify/GenericsTests.java

```diff
@@ -1737,6 +1737,33 @@ public void testRawTypeReceiverCast() {
         .doTest();
   }
 
+  @Test
+  public void testUseOfUnannotatedCode() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.NullMarked;",
+            "import org.jspecify.annotations.NullUnmarked;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  @NullUnmarked",
+            "  static class nullUnmarkedClass<S> {",
+            "      static <T> void m1(T t) {}",
+            "    }",
+            "  @NullMarked",
+            "  static class markedClass {",
+            "    static void testInstantiation() {",
+            "      new nullUnmarkedClass<@Nullable String>();",
```

Comment:
Let's add a comment here (within quotes) indicating that despite the type variable not having a `@Nullable` upper bound, we don't expect an error since the type is `@NullUnmarked`